### PR TITLE
Variation Testing - Target openApiResponse

### PR DESCRIPTION
**Pull request by thim81**
Variation Testing - Added the option to target the expected openApiResponse.

### CODE REVIEW

File: src/application/VariationWriter.ts [hunk 1]
```diff
@@ -107,11 +116,18 @@ export class VariationWriter {
     }
 
     if (oaOperation && tests?.contractTests) {
+      // Set target OpenAPI response
+      let targetOaResponse = variation?.openApiResponse
+      if (!variation?.openApiResponse && variationMeta?.openApiResponse) {
```
Comment (nicklloyd) [inline src/application/VariationWriter.ts:121]: can use `targetOaResponse` here

Comment (nicklloyd) [reply src/application/VariationWriter.ts:121]: just seems weird we check for it and then overwrite it, but maybe I'm missing context

Comment (nicklloyd) [reply src/application/VariationWriter.ts:121]: ah nvm - I get it

Comment (thim81) [reply src/application/VariationWriter.ts:121]: I need to check, if "variation.openApiResponse" is NOT set, then I'm allowed to take the "variationMeta?.openApiResponse" as "targetOaResponse"

Comment (thim81) [reply src/application/VariationWriter.ts]: Corrected

**Pull request merged**